deps/rabbit/test/rabbit_stream_queue_SUITE.erl

```diff
@@ -249,6 +249,12 @@ init_per_group1(Group, Config) ->
                          Config2, stream_queue),
             case EnableFF of
                 ok ->
+                    if Clustered ->
+                           rabbit_ct_broker_helpers:enable_feature_flag(
+                             Config2, stream_update_config_command);
+                       true ->
+                           ok
+                    end,
```

Comment:
I don't think you need to verify if it's clustered or not.

However, you need to verify if the feature flag was successfully enabled. If it failed, you can return something like `{skip, "Failed to enable `stream_update_config_command`"}`. Otherwise, the testcase may fail later.

Reply:
If we enable this for the unclustered tests when they later try to cluster they will now fail (in mixed mode) which is why I don't enable it for all. The tests are written such they pass anyway, we just do things in another way internally.